website/docs/components/form/input-group/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,114 @@
+## Usage
+
+An Input Group is intentionally published as an agnostic container element that wraps one or more inputs and actions, making the relevant use cases broad.
+
+!!! Info
+
+The way in which these components are published and assembled together are fundamenally different between the Figma Ember components.
```

Comment:
"between the Figma Ember components." ??

Reply:
Should be "Figma and Ember compoennts"